format/spec.md

```diff
@@ -1480,6 +1494,9 @@ This serialization scheme is for storing single values as individual binary valu
 | **`struct`**                 | Not supported                                                                                                |
 | **`list`**                   | Not supported                                                                                                |
 | **`map`**                    | Not supported                                                                                                |
+| **`geometry`**               | A single point, encoded as a {x, y, optional z, optional m} concatenation of its 8-byte little-endian IEEE 754 coordinate values, with the optional coordinates encoded as NaN if unset. |
```

Comment:
I don't think this table is the right place to define this encoding because this covers any value and the encoding for arbitrary geometry and geography values is WKB.

For the lower and upper bounds, I suggest adding a new section and documenting the way we store points specifically for both types.

Reply:
We should probably change the text above this table, it specifies this is for lower and upper bounds.

Reply:
Yea I was also thrown off by the above text as specified by Russell.  Changed the text and added a subsection, let me know if it is what you mean.

Reply:
Looks good to me.